regression-test/data/minus_except_test/minus_except.out

```diff
@@ -0,0 +1,3 @@
+-- This file is automatically generated. You should know what you did if you want to edit this
+-- !select --
++06-00  +06-00
```

Comment:
incorrect content format
maybe:
`[+06-00, +06-00]`
or you can try
`sh run-regression-test --run -s minus_except -getOut`

Reply:
This out file is generated by genOut command( I use the version of 2.0.2 )  and local test has passed , shall I change the version to master and regenerate the out file?
What I wonder is why the out file genterated  by local which use command below  can't passed by remote
command I used
![image](https://github.com/apache/doris/assets/53637394/1b748aaa-ab68-410c-82bc-71cf8a19cfca)

Looking forward to your answer @zhiqiang-hhhh

Reply:
I tried on my local, gen out file content is same with yours

Reply:
> I tried on my local, gen out file content is same with yours

so how can I get the result can be passed on remote 😂😂